.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: Python CI Workflow
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Is windows available as an option?

Reply:
Yes, it is. Do you want to run a matrix configuration for all OSes?

Reply:
If we can run for free then why not?😉